spec/smarty_streets_spec.rb

```diff
@@ -75,21 +96,62 @@
           location.zipcode = '95014'
         end
 
-        expect(locations.first.street).to  eq '1 Infinite Loop'
-        expect(locations.first.city).to    eq 'Cupertino'
-        expect(locations.first.state).to   eq 'CA'
+        expect(locations.first.street).to eq '1 Infinite Loop'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.